lib/improver/tests/metadata/test_check_datatypes.py

```diff
@@ -259,26 +253,53 @@ def setUp(self):
 
     def test_pass_cube(self):
         """Test return value for compliant cube"""
-        result = enforce._check_units_and_dtype(self.cube, 'K', np.float32)
+        result = _check_units_and_dtype(self.cube, 'K', np.float32)
         self.assertTrue(result)
 
     def test_fail_cube(self):
         """Test return value for non-compliant cube"""
-        result = enforce._check_units_and_dtype(
+        result = _check_units_and_dtype(
             self.cube, 'degC', np.float32)
         self.assertFalse(result)
 
     def test_pass_coord(self):
         """Test return value for compliant coordinate"""
-        result = enforce._check_units_and_dtype(
+        result = _check_units_and_dtype(
             self.coord, 'm', np.float32)
         self.assertTrue(result)
 
     def test_fail_coord(self):
         """Test return value for non-compliant coordinate"""
-        result = enforce._check_units_and_dtype(self.coord, 'm', np.int32)
+        result = _check_units_and_dtype(self.coord, 'm', np.int32)
         self.assertFalse(result)
 
 
+class Test_check_time_coordinate_metadata(IrisTest):
```

Comment:
Test class moved without changes (to remain consistent with order of functions in module).